extension/my-style.js

```diff
@@ -128,7 +140,14 @@
 
     /* Save styles persistently in local storage. */
     var saveStyles = throttle(function() {
-      localStorage.myStyle = style.innerHTML;
+      var obj = {};
+      obj[storageKey()] = style.innerHTML;
+
+      textarea.setAttribute('syncing', true);
+
+      chrome.storage.sync.set(obj, function() {
+        textarea.setAttribute('syncing', false);
```

Comment:
I can see this being useful, but `syncing` isn't a valid HTML attribute. Make this a custom data-* attribute instead (as the intention of data-* attributes was exactly for this case); i.e. `textarea.setAttribute('data-syncing', false)`.